db/migrate/20230109080950_rename_solidus_paypal_braintree_source_type.rb

```diff
@@ -0,0 +1,16 @@
+class RenameSolidusPaypalBraintreeSourceType < ActiveRecord::Migration[6.1]
+  def up
+    Spree::Payment.where(source_type: 'SolidusPaypalBraintree::Source')
+      .update_all("source_type = 'SolidusBraintree::Source'")
```

Comment:
Concerns that these `.update_all` queries could lock the table for too long? See https://github.com/ankane/strong_migrations#backfilling-data.

Reply:
@waiting-for-dev Okay, I'll go ahead and comment them out with a warning saying that they can lock the tables affected. What do you think?

Reply:
Another option would be implementing them in the way `strong_migrations` recommends. I.e., no transaction, batching and throttling. Pinging @kennyadsl & @elia for more opinions.

Reply:
I can't see why not use the strong migration suggestion immediately. I was about to suggest this code:

```ruby
Spree::Payment.where(...).in_batches.update_all(...)
```

But their (similar) version seems to be even safer and considering more scenarios.